### PR TITLE
React-Redux |Project Setup

### PR DESCRIPTION
**Pull request by errea**
# Description

This PR is to check the basic setup required for a react-redux application. An example is for the book-store project

## Project Requirement 

- [x] Initialise React app.
- [x] Add React Redux 
- [x] Structure your application files using a "feature folder" approach and use the [ducks pattern](https://github.com/erikras/ducks-modular-reduxl) for your Redux files.
- [x] The building blocks of your app should be set as re-usable components.
- [x] Add react-router
- [x] No styling required 

# How Has This Been Tested?
- [x] No test

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: book-store/src/components/books.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Books = () => (
+  <div>
+    In progress...
+  </div>
+);
```
Comment (dasileker) [inline book-store/src/components/books.js]: - [x] As the requirements says you should use  `prop-types` to render the title and the author. not the category

bbb

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @errea**, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: book-store/src/components/books.js (hunk 1 above)
Comment (errea) [reply book-store/src/components/books.js]: Fixed

**Review by bellom — APPROVED** [review]
## Hi @errea  👏  

**Great work making all the necessary corrections from the previous review.**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**